app/models/event.rb

```diff
@@ -9,4 +9,11 @@ class Event < ApplicationRecord
   def has_member?(user)
     members.include?(user)
   end
+
+  def event_status
+    return '' unless self.started_at
```

Comment:
You should not have different state then 3 states from Trello task and this is the 4th one.